.github/workflows/npmPublish.yml

```
@@ -94,7 +95,7 @@ jobs:
           # Create a new commit pointing to the new tree
           commit=$(curl -sf -H "Authorization: token $token" -X POST \
             -d '{
-              "message": "Add multiple files",
+              "message": "Update version to: '"$version"'",
```

Feels like the quotes need escaping here

---

I think the lines blow show that it's not needed, but are you thinking something like this?

```suggestion
              "message": "Update version to: '\"$version\"'",
```

---

hmm, the whole data section is also wrapped in `'` so that also makes it suspicious. Maybe don't wrap it altogether?